clients/database/migrations/20230310154711_add_proxied_request_metrics_indices.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE INDEX IF NOT EXISTS method_name_idx ON proxied_request_metrics(method_name);
```

Comment:
It doesn't matter yet since there is no deployment with a large dataset yet, but it's a good habit to use `CREATE INDEX CONCURRENTLY ...` in order to not block writes while the index is created.

Reply:
split out the migrations so we can create the indices concurrently in separate transactions